samples/tab-staggered-permission/csharp/StaggeredPermission/Pages/auth-end.cshtml.cs

```diff
@@ -28,7 +28,7 @@ public authEnd(IConfiguration configuration)
 	/// </summary>
         public void OnGet()
         {
-            ClientId = _configuration["MicrosoftAppId"];
+            ClientId = _configuration["AzureAd:ClientId"];
```

Comment:
appsetting.json have different name, "MicrosoftAppId" Please correct it.

Reply:
fixed